mempool/src/pool/tests.rs

```diff
@@ -1452,6 +1445,7 @@ async fn rolling_fee(#[case] seed: Seed) -> anyhow::Result<()> {
     );
     let res = mempool.add_transaction(dummy_tx.clone()).await;
 
+    log::debug!("result of first attempt: {:?}", res);
```

Comment:
I would give more context to this if you wanan keep it

Reply:
Done.